src/components/App/index.tsx

```diff
@@ -1,11 +1,15 @@
-import React from 'react';
+import { EgeriaLineage } from '../Lineage';
 
 import './index.scss';
 
 export function App() {
   return (
     <div className="container">
-      <h1>Egeria UI Components</h1>
+      <div>
+        <h1>EgeriaLineage</h1>
+
+        <EgeriaLineage lineage="end-to-end" />
+      </div>
```

Comment:
this is App index not LIneage

Reply:
it is used for sampling the components

Reply:
OK! in the future will have a list of components with separate links